gnlse/gnlse.py

```diff
@@ -163,26 +164,26 @@ def __init__(self, setup):
                                        self.N / 2
                                        ) / (self.N * (self.t[1] - self.t[0]))
         # Central angular frequency [10^12 rad]
-        w_0 = (2.0 * np.pi * c) / setup.wavelength
-        self.Omega = self.V + w_0
+        self.w_0 = (2.0 * np.pi * c) / setup.wavelength
```

Comment:
why do you think it is important to introduce w0 as a class property?

Reply:
w0 is needed to properly calculate wavelengths. Consequently, we need w0 in Solution for plotting functions.